.github/workflows/mvn-verify-check.yml

```diff
@@ -54,7 +54,8 @@ jobs:
         run: |
           set -x
           depsSHA1=$(. .github/workflows/mvn-verify-check/get-deps-sha1.sh 2.12)
-          cacheKey="${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}-${{ github.event.pull_request.base.ref }}-${depsSHA1}"
+          hashfile=$(echo ${{ hashFiles('**/pom.xml') }} | cut -c1-8)
```

Comment:
Instead of truncating hashFile why not feed all the inputs to md5sum as in https://github.com/NVIDIA/spark-rapids/pull/12237/files#diff-4e6d49cc27e982e1413522f21efeb5632004b1b911cbbdfc8a53af561ee4ad92R53 and then truncate if it is really necessary to truncate it

Reply:
Good point, now we just use the first 8 chars of md5sum of raw hash value & deps sha1.

Reply:
Latest test run: https://github.com/YanxuanLiu/spark-rapids/actions/runs/13558189224/job/37896469348?pr=26#step:4:51